specs/signing-scheme.md

```diff
@@ -22,24 +22,27 @@ Generally it covers the following aspects, but can be extended to other aspects
 1. Mechanism to establish trust in end users and other entities (like CAs).
 1. Set of guarantees available to the verifier apart from integrity and authenticity.
 
-## Signing Scheme
+> [!IMPORTANT]
+> A signature envelope is restricted to one signing scheme.
 
-Notary Project currently defines the following Signing Schemes.
+When the Notary Project incorporates support for a new Signing Scheme
+  * For signatures generated using the new signing scheme, it is imperative to update existing signature verification software to newer versions. Failed to so will result in these signatures verification failure. However, signatures generated using older signing schemes will continue to be verified correctly.
+  * The language of the [Notary Project trust policy](./trust-store-trust-policy.md) in *trustpolicy.json* MAY have breaking changes to support newer concepts/configuration elements introduced by the new signing scheme.
+  The breaking changes are addressed by introducing new major version in the versioned *trustpolicy.json*.
```

Comment:
I think the breaking changes are not really addressed as the breaking changes are still there compared with older version. Bumping up to a major version is a release strategy that indicates the breaking changes are acceptable.

```suggestion
  The `version` in *trustpolicy.json* will be bumped up to a major version in case any breaking changes introduced.
```

Reply:
This PR is on `timestamping` only, the marked change here is just a relocation, no content change is involved here. Maybe we could update it in a future PR.